### PR TITLE
Feature/day 3

### PR DESCRIPTION
**Pull request by uditpratap-successive**

### CODE REVIEW

File: day.2/ques2.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+//Program to take the input from the user's name ,age, phone_number and print it output.
+
+let user_name=promt("Enter the name");
+let age=promt("What is your age");
+let phone_number=promt("please enter your phone number");
+
```
Comment (saurabh-successive92) [inline day.2/ques2.js:6]: Please remove prompt

Comment (uditpratap-successive) [reply day.2/ques2.js:6]: Changes has been done.

File: day.3/Ques2_c.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+1 2 3
+1 2
+1
+1 2
+1 2 3
+1 2 3 4
+1 2 3 4 5
+*/
+
+
```
Comment (saurabh-successive92) [inline day.3/Ques2_c.js:13]: Please reduce the number of loops

File: day.3/ques3.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+      *  *
+    *  *  *
+  *  *  *  *
+ *  *  *  *  *
+  *  *  *  *
+    *  *  *
+     *  *
+       *
+*/
+function createDimondShape(size) {
```
Comment (saurabh-successive92) [inline day.3/ques3.js:12]: Please reduce the number of loops

File: day.3/ques4.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+const arr2=[];
+for(let i=1;i<100;i++){
+    if(i%2===0){
+        arr.push(i);
+    }
+    else {
+        arr2.push(i);
+    }
+}
+console.log(arr);
```
Comment (saurabh-successive92) [inline day.3/ques4.js:13]: Please check this 
not getting the expected result

**Pull request merged**